server_tests/test/clientTests.js

```diff
@@ -1528,50 +1539,28 @@ describe("RUST API TEST SUITE", function() {
                     })
                     .then( function onSuccess (res) {
                         Comparison.bublStopOutput(res);
-                    }, shouldNotGetHere);
+                        deferred.resolve();
+                    })
+                    .catch(deferred.reject);
```

Comment:
I was thinking, can I use `.catch(wrapError)` for this? Since I just need an error handler at the end of the promise chain